resolwe/flow/executors/__main__.py

```diff
@@ -0,0 +1,18 @@
+"""Main standalone execution stub, used when the executor is actually run."""
+
+import sys
+from importlib import import_module
+
+from .run import settings
+
+
+def run_executor():
+    """Start the actual execution; instantiate the executor and run."""
```

Comment:
Can you document how this execution stub is called (e.g. what are the arguments)?